diff --git i/config.def.h w/config.def.h
index 5cd37f7..70463b5 100644
--- i/config.def.h
+++ w/config.def.h
@@ -1,22 +1,33 @@
 /* See LICENSE file for copyright and license details. */
 
 /* appearance */
-static const unsigned int borderpx  = 1;        /* border pixel of windows */
+static const unsigned int borderpx  = 2;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int swallowfloating    = 0;        /* 1 means swallow floating windows by default */
 static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
 static const char *fonts[]          = { "monospace:size=20" };
 static const char dmenufont[]       = "monospace:size=20";
-static const char col_gray1[]       = "#222222";
-static const char col_gray2[]       = "#444444";
-static const char col_gray3[]       = "#bbbbbb";
-static const char col_gray4[]       = "#eeeeee";
-static const char col_cyan[]        = "#005577";
+static const char col_black[]       = "#152126";
+static const char col_red[]         = "#b90440";
+static const char col_green[]       = "#07b307";
+static const char col_yellow[]      = "#d7ac00";
+static const char col_blue[]        = "#876bd6";
+static const char col_purple[]      = "#db498b";
+static const char col_cyan[]        = "#41c7d1";
+static const char col_white[]       = "#c0b1c2";
+static const char col_hi_black[]    = "#5f5e66";
+static const char col_hi_red[]      = "#ff246d";
+static const char col_hi_green[]    = "#00ed00";
+static const char col_hi_yellow[]   = "#ffd324";
+static const char col_hi_blue[]     = "#7442ff";
+static const char col_hi_purple[]   = "#ff2088";
+static const char col_hi_cyan[]     = "#13efff";
+static const char col_hi_white[]    = "#e1e1e6";
 static const char *colors[][3]      = {
 	/*               fg         bg         border   */
-	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+	[SchemeNorm] = { col_white   , col_black, col_hi_black },
+	[SchemeSel]  = { col_hi_white, col_blue , col_white    },
 };
 
 /* tagging */
